public/keymaps/p/pom_keyboards_tnln95_default.json

```diff
@@ -0,0 +1,22 @@
+{
+  "keyboard": "pom_keyboards/tnln95",
+  "keymap": "default",
+  "commit": "cf141c00d636b67179f7b9436284fbd0b7f6def2",
+  "layout": "LAYOUT_ansi",
+  "layers": [
+    [
+        "KC_PMNS", "KC_PAST", "KC_PSLS", "KC_BSPACE", "KC_ESC",    "KC_GRAVE", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINUS", "KC_EQUAL",     "KC_BSPACE",    "KC_HOME",
+        "KC_PPLS", "KC_P7",   "KC_P8",   "KC_P9", "KC_NUMLOCK",      "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_LBRC", "KC_RBRC",    "KC_BSLASH",       "KC_PGUP",
+                 "KC_P4",   "KC_P5",   "KC_P6", "CMD_COPY",          "KC_CAPS", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCOLON", "KC_QUOTE",       "KC_ENTER",     "KC_PGDN",
+        "KC_PENT", "KC_P1",   "KC_P2",   "KC_P3", "CMD_PASTE",           "KC_LSHIFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMMA", "KC_DOT", "KC_SLASH",   "KC_RSHIFT",   "KC_UP", "KC_END",
+                 "KC_PDOT",    "KC_P0",      "KC_COMMA" ,       "KC_LCTRL", "KC_LGUI", "KC_LALT",                "KC_SPACE",              "KC_RALT",   "MO(_FNC)",   "KC_LEFT", "KC_DOWN", "KC_RIGHT"
+    ],
+    [
+        "_______", "_______",   "_______",   "_______", "_______",   "_______",   "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",   "KC_F11",   "KC_F12",        "KC_DEL",    "KC_PSCR",
+        "_______", "_______",   "_______",   "_______", "_______",      "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "RGB_TOG", "RGB_RMOD", "RGB_MOD",     "KC_INS",      "KC_VOLU",
+                 "_______",   "_______", "_______", "_______",          "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",         "BL_TOGG",       "KC_VOLD",
+        "_______", "_______",   "_______",   "_______", "_______",           "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "BL_DEC", "BL_INC", "BL_BRTG",         "_______",        "_______", "KC_MUTE",
+                 "_______",       "_______",        "_______" ,       "_______",   "_______",   "_______",                             "_______",                      "_______",     "MO(_FNC)",    "_______", "_______", "_______"

```

Comment:
Updates the keymap for the layout macro change and the keymap update on qmk_firmware repo, plus a couple of bug fixes.

```suggestion
      "KC_CALC", "KC_PAST", "KC_PSLS", "KC_BSPC", "KC_ESC",          "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSLS", "KC_PSCR", "KC_HOME",
      "KC_PMNS", "KC_P7",   "KC_P8",   "KC_P9",   "KC_NLCK",         "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSPC", "KC_PGUP",
      "KC_PPLS", "KC_P4",   "KC_P5",   "KC_P6",   "LCTL(KC_INS)",    "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",  "KC_PGDN",
      "KC_EQL",  "KC_P1",   "KC_P2",   "KC_P3",   "LSFT(KC_INS)",    "KC_LSFT",            "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "KC_UP",   "KC_END",
      "KC_PENT", "KC_PDOT", "KC_P0",   "KC_P0",   "KC_COMM",         "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",             "KC_RALT", "MO(1)",   "KC_RCTL", "KC_LEFT", "KC_DOWN", "KC_RGHT"
    ],
    [
      "_______", "_______", "_______", "_______", "_______",    "_______", "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",   "KC_F12",  "KC_INS",  "KC_DEL",  "_______",
      "_______", "_______", "_______", "_______", "_______",    "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "RGB_TOG", "RGB_RMOD", "RGB_MOD", "_______", "KC_VOLU",
      "_______", "_______", "_______", "_______", "_______",    "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",             "BL_TOGG", "KC_VOLD",
      "_______", "_______", "_______", "_______", "_______",    "_______",            "_______", "_______", "_______", "_______", "_______", "_______", "_______", "BL_DEC",  "BL_INC",  "BL_BRTG",  "_______", "_______", "KC_MUTE",
      "_______", "_______", "_______", "_______", "_______",    "_______", "_______", "_______",                       "_______", "_______", "_______",            "_______", "MO(1)",   "_______", "_______", "_______", "_______"
```